### ISSUE TITLE
Issues about switch off the update G

### ISSUE THREAD

**Issue by wtybest** [XingangPan/deep-generative-prior#10]
When I switch off the options --update_G, to perform the reconstruction, which means only optimize the latent code, the result seems far from satisfactory, even for imagenet val datasets.
![image](https://user-images.githubusercontent.com/55389940/113243369-7602e780-92e5-11eb-91aa-ce38fb64a75f.png)
In my understanding, optimizing just for latent code might not be perfect, but at least it shouldn't be this bad.
Is that consistent with what you tried?

**Comment by XingangPan** [comment]
@wty-ustc Yes, the result is consistent with what I tried.
Optimizing just for latent code would produce very inaccurate reconstruction for BigGAN, even if you enlarge the learning rate. That's why we need to optimize the generator too.

**Comment by wtybest** [comment]
Thank you for your response!&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "XingangPan/deep-generative-prior"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年4月1日(星期四) 下午3:05
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [XingangPan/deep-generative-prior] Issues about switch off the update G (#10)





 
@wty-ustc Yes, the result is consistent with what I tried.
 Optimizing just for latent code would produce very inaccurate reconstruction for BigGAN, even if you enlarge the learning rate. That's why we need to optimize the generator too.

**Comment by zhangxiangyu19** [comment]
it's the same with my test. How do other generators do it, is it because the training set is smaller?
